src/services/transcript-parser.js

```diff
@@ -371,13 +389,33 @@ export async function parseTranscriptData(url, format, canvasIndex, parseMetadat
   }
 }
 
+/**
+ * Convert a list of TextualBody annotations for a Canvas in a given IIIF Manifest,
+ * into a format that supports the transcript component cue display
+ * @param {Array} annotations 
+ * @returns {Array}
+ */
 function createTData(annotations) {
   if (annotations?.length === 0) return [];
   let tData = [];
+
+  // Build text from an array of TextualBody in an annotation
+  let buildText = (texts) => {
+    if (!texts?.length === 0) return '';
```

Comment:
I don't understand the logic in this line.  When I test in the browser console I don't see how this can ever be true.  Can you please explain?